app/views/landing_page/index.html.erb

```diff
@@ -1 +1,2 @@
-<h1>Hello World</h1>
\ No newline at end of file
+<h1>Hello World</h1>
+
```

Comment:
Be careful on creating commit noise (files changed that have no code changes). I personally use sourcetree (there are others too) to check over my changes while I am committing to make sure I discard any unintentional lines. 

No need to change here though, just going forward =)